Arista/ConfigFiles/telegraf-qbit-storage.conf

```diff
@@ -0,0 +1,178 @@
+# Telegraf Configuration
```

Comment:
yes, that would make more sense since we are explicitly using scylla path in the config
I'll change it